API.md

```diff
@@ -1275,6 +1275,69 @@ curl http://localhost:<port>/irods-http-api/<version>/query \
 }
 ```
 
+### add_specific_query
+
+Adds a specific query.
+
+#### Request
+
+```bash
+curl http://localhost:<port>/irods-http-api/<version>/query \
+    -H 'Authorization: Bearer <token>' \
+    --data-urlencode 'op=add_specific_query' \
+    --data-urlencode 'name=<string>' \ # The name of specific query.
+    --data-urlencode 'sql=<string>' # The SQL to map to the specific query name.
+```
+
+#### Response
+
+If an HTTP status code of 200 is returned, the body of the response will contain JSON. Its structure is shown below.
+
+```js
+{
+    "irods_response": {
+        "status_code": 0
+        "status_message": "string" // Optional
+    },
+    "rows": [
+        ["string", "string", "string"],
+
+        // Additional rows ...
+    ]
```

Comment:
I see that these are in the other responses, but it's not clear to me what this will contain. Is it just the rows that were updated?

Reply:
That's a copy-paste issue. Will remove.

Reply:
Done